app/controllers/standards_imports_controller.rb

```diff
@@ -24,7 +24,26 @@ def show
 
   private
 
-  def create_params
+  def standards_import_params
+    if current_user&.admin?
+      admin_create_params
+    else
+      user_create_params
+    end
+  end
+
+  def user_create_params
     params.require(:standards_import).permit(:name, :email, :organization, :notes, files: [])
   end
+
+  def admin_create_params
+    params.require(:standards_import).permit(
+      :name,
+      :email,
+      :organization,
+      :notes,
+      :public_document,
+      files: []
+    )
+  end
```

Comment:
I'm late to the game here, but a change for the future could be that we set up a Pundit policy for StandardsImport and use the policy to define the permitted parameters. See [SourceFile policy](https://github.com/ApprenticeshipStandardsDotOrg/ApprenticeshipStandardsDotOrg/blob/main/app/policies/source_file_policy.rb#L24-L29) and [controller](https://github.com/ApprenticeshipStandardsDotOrg/ApprenticeshipStandardsDotOrg/blob/main/app/controllers/admin/source_files_controller.rb#L21-L23) as an example.